rideshare/Controllers/SettingsViewController.swift

```diff
@@ -71,14 +72,16 @@ class SettingsViewController: UIViewController, UITableViewDelegate, UITableView
         if indexPath.row == 0 {
             let cell = tableView.dequeueReusableCell(withIdentifier: "settingsProfileCell") as! SettingsProfileCell
             
-            // TODO: set fname and lname based on PFUser
-            let fname = "First"
-            let lname = "Last"
-            // end update
-            
             // TODO: update profile picture using profile pic of user
```

Comment:
you can remove this line now!

Reply:
Done and should now be ready to be merged in